cli/ballerina-cli/src/main/resources/cli-help/ballerina-shell.help

```diff
@@ -0,0 +1,33 @@
+NAME
+       ballerina-shell - Run ballerina interactive REPL
+
+SYNOPSIS
+       bal shell [-d|--debug] [-f|--force-dumb]  [-t|--time-out]
+
+
+DESCRIPTION
+       Shell runs a REPL instance of ballerina to enable users to run small
+       snippets to code to check their outputs. Please not that this is an
+       experimental feature.
+
+       Debug messages can be enabled via -d parameter.
+
+OPTIONS
+       -d
```

Comment:
We don't usually use short options except for maybe help. Given that this is for REPL, if we think they'll be used often, I guess it's OK to include them. Or for consistency, we can stick to the long options.

Reply:
Should I leave short options as is then?

Reply:
Yeah this is OK for the time-being.